pypeit/spectrographs/gemini_gnirs.py

```diff
@@ -224,27 +226,17 @@ def setup_arcparam(self, arcparam, fitstbl=None, arc_idx=None,
         """
         # ToDo need to parse the sigdetect parameter to be here for detect_lines function in arc.py
         #      I force change sigdetect=5 for GNIRS.
+        arcparam['lamps'] = ['OH_GNIRS'] # Line lamps on
```

Comment:
I should think you can remove `arcparam` now.  Right?

Reply:
Yeah, I've left some of that code in though.